.travis.yml

```diff
@@ -276,6 +276,20 @@ jobs:
       before_script:
         - bash ./tests/travis/install-postgres-10.sh
 
+    - stage: Test
+      php: 7.1
+      env: DB=sqlite DEPENDENCIES=low
+      install:
+        - travis_retry composer update --prefer-dist --prefer-lowest
+
+    - stage: Test
+      if: type = cron
+      php: 7.2
+      env: DB=sqlite DEPENDENCIES=dev
+      install:
+        - composer config minimum-stability dev
```

Comment:
From https://github.com/doctrine/doctrine2/pull/6887#issuecomment-351502322:

> Testing against dev dependencies is highly unreliable and non-deterministic.

Hopefully, someone will have enough time to triage the failures. BTW, where on Travis CI can I find the build results? I see https://github.com/doctrine/doctrine2/pull/6887 is already merged.

Reply:
Yes, that is (still) true, that's why it's not good for PRs. But it's good to spot bugs/incompatibilities early. It won't be noisy during PR/merge/push.

> where on Travis CI can I find the build results?

It's listed with other builds, merked with "CRON" label:
https://travis-ci.org/doctrine/doctrine2/builds
https://travis-ci.org/doctrine/doctrine2/jobs/322719675
It's not enabled for DBAL (yet).

Reply:
FYI: First CRON build is there and passed: https://travis-ci.org/doctrine/dbal/builds